templates/common/openstack/units/provider-config-symlink.yaml

```diff
@@ -0,0 +1,12 @@
+name: "provider-config-symlink.service"
+enabled: true
+contents: |
+  [Unit]
+  Description=Create a symlink '/etc/kubernetes/cloud.conf' -> '/etc/kubernetes/cloud-config'
```

Comment:
Surely this would need to be `Before=kubelet.service`?

Also this service will fail if the symlink already exists (on reboot)...so you'd want `ConditionPathExists=!/etc/kubernetes/cloud-config` or so.

Hmm.  It's too bad our internal Ignition templating doesn't support symlinks, then this would be a lot more obvious.

Reply:
yeah, I agree. Before=kubelet.service is not really necessary, because CP starts well and fails only when we try to attach a volume from cinder to a container. But I think it wouldn't be bad to have this dependency as well.